cockatrice/src/player.cpp

```diff
@@ -1758,7 +1761,11 @@ void Player::createCard(const CardItem *sourceCard, const QString &dbCardName, b
     } else {
         cmd.set_annotation("");
     }
-    cmd.set_destroy_on_zone_change(true);
+    if (conjured) {
+		cmd.set_destroy_on_zone_change(false);
+	} else {
+		cmd.set_destroy_on_zone_change(true);
+	}
```

Comment:
You can simplify this if/else block

```suggestion
    cmd.set_destroy_on_zone_change(conjured);
```